shell/browser/browser_mac.mm

```diff
@@ -374,17 +374,24 @@ void RemoveFromLoginItems() {
 }
 
 void Browser::ShowAboutPanel() {
-  NSDictionary* options = DictionaryValueToNSDictionary(about_panel_options_);
+  NSMutableDictionary* mutable_options =
+      [DictionaryValueToNSDictionary(about_panel_options_) mutableCopy];
```

Comment:
`mutableCopy` returns a copy, the result has also to be managed with `autorelease` or `base::scoped_nsobject`.

And it is better to put the copy inside the `if (credits != nil)` so the copy only happens when necessary.